docs/content/flags.md

```diff
@@ -116,6 +116,7 @@ These flags are available for every command.
       --rc-job-expire-interval duration      interval to check for expired async jobs (default 10s)
       --rc-key string                        SSL PEM Private key
       --rc-max-header-bytes int              Maximum size of request header (default 4096)
+      --rc-min-tls-version string            the minimum TLS version that is acceptable
```

Comment:
Minor nitpick: The description here is normally the exact text from command help (and capitalized first letter, though I realise not all existing have that):
```
flags.StringVarP(flagSet, &Opt.MinTLSVersion, prefix+"min-tls-version", "", Opt.MinTLSVersion, "Minimum TLS version")
```

Reply:
Minor nitpick addressed. :)

Reply:
Good. If you could make the first letter capitalized that would better match the majority of others, and the preferred style.

(
I realize there are some other rc flags with lowercase first letter, but those should be also be corrected correspondingly:

```
      --rc-htpasswd string                 htpasswd file - if not provided no authentication is done
      --rc-job-expire-duration duration    expire finished async jobs older than this value (default 1m0s)
      --rc-job-expire-interval duration    interval to check for expired async jobs (default 10s)
      --rc-realm string                    realm for authentication (default "rclone")
```
)

Also, don't know if we should just remove the initial `the`, and make it start with `Minimum`? Just a suggestion... :) But seems most others are like that: `Time to wait...`, `Max total size...`, etc.

Reply:
Good idea, done.